src/App.tsx

```diff
@@ -50,6 +50,8 @@ import createTheme from 'utils/createCustomTheme'
 import graphqlClient from 'utils/graphql'
 import { RewardsProvider } from './contexts/Rewards'
 
+const DISCORD_LINK = 'https://discord.com/invite/74xXr57XHT'
```

Comment:
@controtie We should consider where we want to put constants like this. `App.tsx` seems like a suboptimal place.

Reply:
Good point. It looks like `src/constants` already exists. What are your thoughts on a `src/constants/externalLinks.ts` file?

Reply:
That seems like a good place, but I defer to @controtie.

Reply:
src/constants/externalLinks sounds great!

Reply:
Cool! Just pushed a change adding that file + a small stylistic cleanup